lua/lush_theme/bluloco.lua

```diff
@@ -422,6 +422,55 @@ local theme = lush(function(injected_functions)
     -- BufferLineInfoSelected { fg = t.info, sp = t.primary, gui = "underline" },
     -- BufferLineHintSelected { fg = t.hint, sp = t.primary, gui = "underline" },
 
+    -- BarBar
+
+    BufferCurrent { fg = t.fg, bg = t.bg },
+    BufferCurrentIndex { BufferCurrent },
+    BufferCurrentIcon { BufferCurrentIndex },
+    BufferCurrentMod { BufferCurrent },
+    BufferCurrentSign { fg = t.keyword, bg = BufferCurrent.bg },
+    BufferCurrentTarget { fg = shade(t.error, 40), bg = BufferCurrent.bg },
+    BufferCurrentWARN { fg = DiagnosticWarn.fg, bg = BufferCurrent.bg },
+    BufferCurrentINFO { fg = DiagnosticInfo.fg, bg = BufferCurrent.bg },
+    BufferCurrentERROR { fg = DiagnosticError.fg, bg = BufferCurrent.bg },
+    BufferCurrentHINT { fg = DiagnosticHint.fg, bg = BufferCurrent.bg },
+
+    BufferInactive { fg = t.shade40, bg = t.bgFloat },
+    BufferInactiveIcon { BufferInactive },
+    BufferInactiveIndex { BufferInactive },
+    BufferInactiveMod { BufferInactive },
+    BufferInactiveSign { BufferInactive },
+    BufferInactiveTarget { fg = shade(BufferInactive.fg, 70), bg = BufferInactive.bg },
+    BufferInactiveWARN { BufferCurrentWARN, bg = BufferInactive.bg },
+    BufferInactiveINFO { BufferCurrentINFO, bg = BufferInactive.bg },
+    BufferInactiveERROR { BufferCurrentERROR, bg = BufferInactive.bg },
+    BufferInactiveHINT { BufferCurrentHINT, bg = BufferInactive.bg },
+
+    BufferVisible { BufferCurrent, bg = t.bgFloat },
+    BufferVisibleIndex { BufferVisible },
+    BufferVisibleIcon { BufferVisibleIndex },
+    BufferVisibleMod { BufferVisible },
+    BufferVisibleSign { BufferVisible },
+    BufferVisibleTarget { fg = shade(BufferVisible.fg, 70), bg = BufferVisible.bg },
+    BufferVisibleWARN { BufferInactiveWARN },
+    BufferVisibleINFO { BufferInactiveINFO },
+    BufferVisibleERROR { BufferInactiveERROR },
+    BufferVisibleHINT { BufferInactiveHINT },
+
+    BufferAlternate { BufferInactive, fg = t.tag },
+    BufferAlternateIndex { BufferAlternate },
+    BufferAlternateIcon { BufferAlternateIndex },
+    BufferAlternateMod { BufferInactiveMod },
+    BufferAlternateSign { BufferInactiveSign },
```

Comment:
blue would be just too close to the actual current one. how about purple for alternate sign?
```suggestion
    BufferAlternateSign { BufferInactiveSign, fg = t.constant },
```

Reply:
ok, here's how that looks now;
![image](https://user-images.githubusercontent.com/19446243/209031862-5f7be98f-49ed-446c-94d4-18aa833a9330.png)

Reply:
I like it, thanks :)